.gitignore

```diff
@@ -12,3 +12,6 @@ thumbs.db
 
 # models
 *.onnx
+
+model/
+assets/images/20**
```

Comment:
Are these files specific to your setup? If so, you should put these in `.git/info/exclude` rather than the shared `.gitignore`